libs/common/src/lib/notifications/notification.service.ts

```diff
@@ -22,8 +24,15 @@ export class NotificationService {
   ) {
     combineLatest([this.sdk.currentAccount, this.sdk.currentKb])
       .pipe(
+        tap(([, kb]) => {
```

Comment:
If I logout and login again, it seems the notifications connection is lost. This also happens when navigating to the user profile. Are these edge cases worth fixing ?

Reply:
yes they do, thank you for raising those issues, I'll fix them before merging :)